packages/components/src/components/positions-drawer-card/positions-drawer-card.tsx

```diff
@@ -88,6 +88,12 @@ const PositionsDrawerCard = ({
         'dc-contract-card--red': Number(profit_loss) < 0 && !result,
     });
 
+    const is_unsupported =
```

Comment:
We can't rely on old `is_unsupported` flag, coming from BE anymore if we want to show information for forward starting contract. Instead of it, we should check if forward starting hasn't started yet and if this contract is in unsupported config.